platform/ui/src/components/Button/Button.jsx

```diff
@@ -26,60 +30,99 @@ const disabledClasses = {
 
 const variantClasses = {
   text: {
-    default: 'hover:bg-gray-200 text-black',
-    primary: 'hover:bg-blue-100 text-blue-900',
-    secondary: 'hover:bg-blue-100 text-blue-300',
+    default:
+      'text-custom-aquaBright hover:bg-custom-aquaBright hover:text-white active:opacity-80 focus:bg-custom-aquaBright focus:text-white',
+    primary:
+      'text-custom-blue hover:bg-custom-blue hover:text-white active:opacity-80 focus:bg-custom-blue focus:text-white',
+    secondary:
+      'text-custom-violetPale hover:bg-custom-violetPale hover:text-white active:opacity-80 focus:bg-custom-violetPale focus:text-white',
+    white:
+      'text-white hover:bg-white hover:text-black active:opacity-80 focus:bg-white focus:text-black',
   },
   outlined: {
-    default: 'border border-black text-black hover:bg-gray-200',
-    primary: 'border border-blue-900 text-blue-900 hover:bg-blue-100',
-    secondary: 'border border-blue-300 text-blue-300 hover:bg-blue-100',
+    default:
+      'border bg-trasparent border-custom-aquaBright text-custom-aquaBright hover:opacity-80 active:opacity-100 focus:opacity-80',
+    primary:
+      'border bg-transparent border-custom-blue text-custom-blue hover:opacity-80 active:opacity-100 focus:opacity-80',
+    secondary:
+      'border bg-transparent border-custom-violetPale text-custom-violetPale hover:opacity-80 active:opacity-100 focus:opacity-80',
+    white:
+      'border bg-transparent border-white text-white hover:opacity-80 active:opacity-100 focus:opacity-80',
   },
   contained: {
-    default: 'bg-black hover:bg-gray-800 text-white',
-    primary: 'bg-blue-900 hover:bg-blue-800 text-white',
-    secondary: 'bg-blue-300 hover:bg-blue-500 text-black',
+    default:
+      'bg-custom-aquaBright text-white hover:opacity-80 active:opacity-100 focus:opacity-80',
+    primary:
+      'bg-custom-blue text-white hover:opacity-80 active:opacity-100 focus:opacity-80',
+    secondary:
+      'bg-custom-violetPale text-white hover:opacity-80 active:opacity-100 focus:opacity-80',
+    white:
+      'bg-white text-black hover:opacity-80 active:opacity-100 focus:opacity-80',
   },
 };
 
 const sizeClasses = {
-  small: 'py-1 px-2 text-sm',
-  medium: 'py-2 px-4 text-base',
-  large: 'py-3 px-6 text-lg',
+  small: 'py-2 px-2 text-base',
```

Comment:
not sure if I like the idea of having small as kind of default size using "base"..

Reply:
The default font size for Button is `16px`/`1rem`, equivalent to what we currently have in the theme config value for `text-lg`. Do you think it would be better to have `16px` for `text-base` instead of `text-lg`? We can change these values here:

https://github.com/OHIF/Viewers/blob/86e1faa0407817d319690b04f90b299bc90b6861/platform/ui/tailwind.config.js#L273-L284

Any idea?